metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorDatabaseService.java

```diff
@@ -81,7 +81,7 @@ public void create(@Nonnull final ConnectorContext requestContext, @Nonnull fina
         } catch (MetaException exception) {
             throw new InvalidMetaException(databaseInfo.getName(), exception);
         } catch (TException exception) {
-            throw new ConnectorException(databaseInfo.getName().toString(), exception);
+            throw new ConnectorException(databaseInfo.getName().toString() + " create error", exception);
```

Comment:
Can we change the message to - String.format("Failed creating hive database %s", databaseInfo.getName()). Make similar changes to other connector exceptions below.